src/app/shared/theme-chooser/theme-chooser.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, ViewEncapsulation, ChangeDetectionStrategy} from '@angular/core';
+import {StyleManager} from '../style-manager/style-manager';
+
+@Component({
```

Comment:
We should make the theme chooser `aria-hidden="true"` and ensure that the button is not in the tab order since a screen-reader user won't get any benefit from changing the app colors.